twine/utils.py

```diff
@@ -157,14 +157,14 @@ def check_status_code(response, verbose):
               "https://packaging.python.org/guides/migrating-to-pypi-org/ ")
     try:
         response.raise_for_status()
-    except HTTPError as err:
+    except requests.exceptions.RequestException as exc:
```

Comment:
`raise_for_status` should only ever raise an `HTTPError`. This seems like an unnecessary change.